Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessAsOwnerTests.cs

```diff
@@ -59,6 +61,21 @@ public void Throws_when_deletion_process_is_in_wrong_status()
         acting.Should().Throw<DomainException>().Which.Code.Should().Be("error.platform.validation.device.deletionProcessIsNotInRequiredStatus");
     }
 
+    [Fact]
+    public void Triggers_IdentityDeletionProcessStatusChangedDomainEvent_when_Canceling()
```

Comment:
Sometimes you're using the word "Raises" and sometimes "Triggers". I know that we had this mixture in the past already, but we should take the opportunity to unify it. And we should go with "Raises", because that's how it's called in `Entity.cs`.